perf_dashboard/artifacts/templates/artifact.html

```diff
@@ -52,15 +52,17 @@ <h1 class="h3 display"><a href="https://console.cloud.google.com/storage/browser
                 <table class="table table-striped">
                   <thead>
                   <tr>
+                    <th>Date</th>
                     <th>Build</th>
                     <th>GCS Link</th>
                   </tr>
                   </thead>
                   <tbody>
-                  {% for release in cur_release_bundle %}
                   <tr>
-                    <td>{{ release|first }}</td>
-                    <td><a href={{ release|last }} target="_blank">data</a></td>
+                  {% for release in cur_release_bundle %}
```

Comment:
This doesn't look like it will generate the html right. The for loop is inside of the <tr> but adds a </tr> before the end for.

Reply:
this {% for .... %} {% endfor %} is the django template for-loop syntax

Reply:
Unresolving it because I don't think this was addressed. My comment was more about the fact that you start a new row in HTML, add the for, close the row, and then end the for. I think the row should be both started and ended in the for rather than alternating. I.e. 

```
 {% for release in cur_release_bundle %}
 <tr>
 </tr>
 {% endfor %}
```

rather than 

```
 <tr>
 {% for release in cur_release_bundle %}\
 </tr>
 {% endfor %}
```

Reply:
Sorry, i'm confused, I did like what your first code block is, not?

I mean like this:
```
{% for release in master_release_bundle %}
                  <tr>
                    <td>{{ release.0 }}</td>
                    <td>{{ release.1 }}</td>
                    <td><a href={{ release.2 }} target="_blank">data</a></td>
                  </tr>
 {% endfor %}
```

Reply:
I see the following in the PR:
```
 <tr>
                  {% for release in cur_release_bundle %}
                    <td>{{ release.0}}</td>
                    <td>{{ release.1 }}</td>
                    <td><a href={{ release.2 }} target="_blank">data</a></td>
                  </tr>
                  {% endfor %}
```
For two cases, won't this print:
```
<tr>
 <td>{{ release.0}}</td>
 <td>{{ release.1 }}</td>
 <td><a href={{ release.2 }} target="_blank">data</a></td>
</tr>
<td>{{ release.0}}</td>
 <td>{{ release.1 }}</td>
 <td><a href={{ release.2 }} target="_blank">data</a></td>
</tr>
<td>{{ release.0}}</td>
 <td>{{ release.1 }}</td>
 <td><a href={{ release.2 }} target="_blank">data</a></td>
</tr>
```
With the </tr> repeating on every iteration due to the fact that it's inside of the for loop while <tr> is not?  Prior to this PR, the <tr> was inside of the for loop.

Reply:
I'm just suggesting moving line 62:  `{% for release in cur_release_bundle %}` above line 61 `<tr>`. I think we're saying the same thing but for some reason feel like we're looking at something different?

Reply:
Perfect. Thanks :)

Reply:
sorry, yeah, we were checking the different place. updated it!